test/FluentMigrator.Tests/Unit/Builders/Delete/DeleteDataExpressionTests.cs

```diff
@@ -110,10 +111,13 @@ public void CallingIsNullAddsANullColumn()
         [Test]
         public void DefaultMigrationExpressionValidatorShouldReturnErrorWhenTableNameIsNotSpecified()
         {
+            // Ensure error messages are in English
+            CultureInfo.CurrentUICulture = CultureInfo.InvariantCulture;
```

Comment:
Why not just use [Test, Culture("en-US")]

Reply:
I didn't know about this but this seems very handy! Thanks for pointing that out!👍

Reply:
I use `SetUICultureAttribute`, as the validation messages rely on `CurrentUICulture` and not on `CurrentCulture` (tested).

Reply:
thanks. will kick off ci service in a bit

Reply:
Updated to empty string